desktop-app/app/components/ExtensionsManager/index.js

```diff
@@ -167,7 +174,9 @@ export default function ExtensionsManager({triggerNavigationReload}) {
 
           <FormHelperText className={styles.extensionsNotice}>
             Note: Only DevTool extensions will work properly, other general
-            browser extensions may not work as intended.
+            browser extensions may not work as intended. Local devtools
+            extensions are also supported. Instead of ID, use absolute path to
```

Comment:
@prashantpalikhe If possible can you please add an `open` icon near the `help` icon and let the user select the folder by using the file explorer? 

It will a simple change using the inbuilt `dialog` API, check out a sample usage here - https://github.com/manojVivek/responsively-app/blob/master/desktop-app/app/menu.js#L176

Reply:
Updated. Can you check?